spec/system/placements/schools/school_contacts/user_adds_a_placement_contact_spec.rb

```diff
@@ -0,0 +1,143 @@
+require "rails_helper"
+
+RSpec.describe "Adding a placement contact to a school", service: :placements, type: :system do
+  scenario "User adds a placement contact to their school" do
+    given_i_am_signed_in
+
+    when_i_view_my_organisation_details_page
+    then_i_do_not_see_any_placement_contact_details
+
+    when_i_click_on_add_placement_contact
+    and_i_submit_the_form
+    then_i_see_a_validation_error_for_the_email_address
+
+    when_i_fill_in_the_placement_contact_form
+    and_i_submit_the_form
+    then_i_see_the_check_your_answers_page
+
+    when_i_change_the_first_name
+    and_i_submit_the_form
+    then_i_see_the_check_your_answers_page_with_the_updated_first_name
+
+    when_i_change_the_last_name
+    and_i_submit_the_form
+    then_i_see_the_check_your_answers_page_with_the_updated_last_name
+
+    when_i_change_the_email_address
+    and_i_submit_the_form
+    then_i_see_the_check_your_answers_page_with_the_updated_email_address
+
+    when_i_click_on_add_placement_contact
+    then_i_see_my_placement_contact_details
+    and_i_see_a_success_message
+  end
+
+  private
+
+  def given_i_am_signed_in
+    @school = build(:placements_school, with_school_contact: false)
+    sign_in_placements_user(organisations: [@school])
+  end
+
+  def when_i_view_my_organisation_details_page
+    click_on "Organisation details"
+    expect(page).to have_title("Organisation details - Manage school placements - GOV.UK")
+    expect(primary_navigation).to have_current_item("Organisation details")
+    expect(page).to have_h1(@school.name)
+    expect(page).to have_h2("Placement contact")
+    expect(page).to have_h2("School")
+    expect(page).to have_summary_list_row("Name", @school.name)
+    expect(page).to have_summary_list_row("UKPRN", @school.ukprn)
+    expect(page).to have_summary_list_row("URN", @school.urn)
```

Comment:
Given what we talked about this morning, you might want to switch these to hardcoded values throughout this file.

```suggestion
    expect(page).to have_summary_list_row("Name", "Hogwarts")
    expect(page).to have_summary_list_row("UKPRN", "10000001")
    expect(page).to have_summary_list_row("URN", "50005")
```